docs-en/chapter_computational_complexity/summary.md

```diff
@@ -2,48 +2,48 @@
+    Theoretically, the space complexity of a tail recursion function can be optimized to $O(1)$. However, most programming languages (e.g., Java, Python, C++, Go, C#, etc.) do not support auto-optimization for tail recursion, so the space complexity is usually considered as $O(n)$.
 
 !!! question "What is the difference between the terms function and method?"
 
-    A function can be executed independently and all arguments are passed explicitly. A method is associated with an object, is implicitly passed to the object that calls it, and can manipulate data contained in instances of the class.
+    A *function* can be executed independently, and all arguments are passed explicitly. A *method* is associated with an object and is implicitly passed to the object that calls it, and can manipulate data contained in instances of the class.
 
-    Here are a few common programming languages to illustrate.
+    Let's illustrate with a few common programming languages.
 
-    - C is a procedural programming language and has no object-oriented concepts, so it has only functions. However, we can simulate object-oriented programming by creating structures, and the functions associated with structures are equivalent to methods in other languages.
-    - Java and C# are object-oriented programming languages, and blocks of code (methods) are usually included as part of a class. A static method behaves like a function because it is bound to the class and cannot access specific instance variables.
+    - C is a procedural programming language without object-oriented concepts, so it has only functions. However, we can simulate object-oriented programming by creating structures (struct), and the functions associated with structures are equivalent to methods in other languages.
+    - Java and C# are object-oriented programming languages, and blocks of code (methods) are typically part of a class. Static methods behave like a function because it is bound to the class and cannot access specific instance variables.
     - C++ and Python support both procedural programming (functions) and object-oriented programming (methods).
 
 !!! question "Does the figure "Common Types of Space Complexity" reflect the absolute size of the occupied space?"
 
     No, that figure shows the space complexity, which reflects the growth trend, not the absolute size of the space occupied.
    
-    Suppose you take $n = 8$ , you may find that the values of each curve do not correspond to the function. This is because each curve contains a constant term that is used to compress the range of values to a visually comfortable range.
+    For example, if you take $n = 8$ , the values of each curve do not align with the function because each curve contains a constant term used to compress the range of values to a visually comfortable range.
```

Comment:
I'm a bit confused with the expression `$n = 8$`? What does the trailing `$` mean? This may be a question to @krahets

Reply:
@thomasq0 This is the latex syntax of inline-math. All the expressions in `$ $` will be rendered as math equations by LaTex.